actions/storage.js

```diff
@@ -51,3 +51,13 @@ export function actionOnGlobalItemsWithPrefix(prefix, action) {
         data: {prefix, action},
     };
 }
+
+export function cleanLocalStorage() {
+    for (const key of Object.keys(localStorage)) {
+        // Remove all keys added for user profile colours before MM-47782
+        const userProfileColorPattern = /^[a-z0-9.\-_]+-#[a-z0-9]{6}$/i;
+        if (userProfileColorPattern.test(key)) {
+            localStorage.removeItem(key);
+        }
+    }
```

Comment:
One would thing by the function name [clears complete LC](https://developer.mozilla.org/en-US/docs/Web/API/Storage/clear) can we rename it to be more specific

Reply:
I can't come up with a better name for this that isn't overly descriptive, especially if we start removing other stuff from the localStorage on logout, so I think this is good enough.